examples/mobile/HomeApp/js/app.js

```diff
@@ -153,6 +170,15 @@ import Storage from "./clipping-storage.js";
 
 	}
 
+	function updateManifests() {
+		for (const app of appsList) {
+			for (const webclip of app.webClipsList) {
+				manifestsList[webclip.url] = fetch(`${webclip.url}/manifest.json`)
```

Comment:
can we just keep result of json fetch in appsList object rather than on separate manifestList?
We already store additional properties there as isSelected for webclip.
This would be stored/restored from local storage together with apps list and could make 
updateAppsList simpler